src/libs/ReportUtils.js

```diff
@@ -2901,12 +2901,13 @@ function buildOptimisticChatReport(
     welcomeMessage = '',
 ) {
     const currentTime = DateUtils.getDBTime();
+    const isNewlyCreatedWorkspaceChat = chatType === CONST.REPORT.CHAT_TYPE.POLICY_EXPENSE_CHAT && isOwnPolicyExpenseChat;
     return {
         type: CONST.REPORT.TYPE.CHAT,
         chatType,
         hasOutstandingIOU: false,
         isOwnPolicyExpenseChat,
-        isPinned: reportName === CONST.REPORT.WORKSPACE_CHAT_ROOMS.ADMINS,
+        isPinned: reportName === CONST.REPORT.WORKSPACE_CHAT_ROOMS.ADMINS || (reportName !== CONST.TEACHERS_UNITE.POLICY_NAME && isNewlyCreatedWorkspaceChat),
```

Comment:
Why not for the teachers unite policy?

Reply:
I thought that we should pin only newly created workspaces by users. However,  the teachers unite one the users don't create. Should we pin the teachers one as well?

Reply:
Ah, I don't think that scenario would be possible as someone can't create the teachers unite workspace since it already exists. So I don't think we need a special condition for it.

Reply:
Basically, the only scenario where you get optimistic workspace chat reports is if you're the one creating the workspace. So we just want to make sure that when someone creates a workspace, their own workspace chat for that workspace gets pinned.

Reply:
@puneetlath, @rushatgabhane, done - c0c8580.